knox-token-management-ui/token-management/app/token.management.component.ts

```diff
@@ -57,13 +117,29 @@ export class TokenManagementComponent implements OnInit {
     }
 
     fetchAllKnoxTokens(): void {
-        this.fetchKnoxTokens(true);
         this.fetchKnoxTokens(false);
+        this.fetchKnoxTokens(true);
```

Comment:
Is the order important here? Just trying to understand the change.

Reply:
Nope, the order is irrelevant. I just made to change to reflect the order of the tables on te GUI: my tokens first then the impersonation tokens.